user/user-tool/tool/src/webapp/vm/user/chef_users_edit.vm

```diff
@@ -138,7 +138,7 @@ function removeOptionalAttributeBlock(elem) {
 				<label for="user_pw0">
 					$tlang.getString("usecre.vernewpass")
 				</label>
-				<input type="password" name="pw0" id="user_pw0" onblur="USER.verifyPasswordsMatch();" autocomplete="off" />
+				<input type="password" name="pw0" id="user_pw0" onkeyup="USER.verifyPasswordsMatch()" onblur="USER.verifyPasswordsMatch();" autocomplete="off" />
```

Comment:
yes, but still an improvement in seeing the mismatch message when typing in the first box which was the original problem.  This fix is one of the suggestions made by Adam in the Jira.